curl/src/main/scala/org/http4s/curl/http/CurlRequest.scala

```diff
@@ -17,6 +17,7 @@
 package org.http4s.curl.http
 
 import cats.effect._
+import cats.implicits._
```

Comment:
`cats.implicits._` is effectively deprecated, I think `cats.syntax.all._` should be sufficient.